src/Pages/Settings/Addons.elm

```diff
@@ -65,12 +68,116 @@ view : Model -> Document Msg
+                        , settingsToggleBlock "urllib3" "urllib3 is a powerful, user-friendly HTTP client for Python. Much of the Python ecosystem already uses urllib3 and you should too. urllib3 brings many critical features that are missing from the Python standard libraries"
+
+                        , el[ Font.color (rgb255 18 178 231), Font.size 24, Font.light, paddingEach {top = 0, bottom = 30, left = 0, right = 0}](text "xbmc.service")
+                        , settingsToggleBlock "Version Check" "Kodi Version Check only supports a number of platforms/distros as releases may differ between them. For more information visit the kodi.tv website."
+
+                        , el[ Font.color (rgb255 18 178 231), Font.size 24, Font.light, paddingEach {top = 0, bottom = 30, left = 0, right = 0}](text "xbmc.gui.skin")
+                        , settingsToggleBlock "Estouchy" "Skin designed to be used on touchscreen devices like tablets and smartphones"
+                        , settingsToggleBlock "Estuary" "Estuary is the default skin for Kodi 17.0 and above. It attempts to be easy for first time Kodi users to understand and use."
+
+                        , el[ Font.color (rgb255 18 178 231), Font.size 24, Font.light, paddingEach {top = 0, bottom = 30, left = 0, right = 0}](text "xbmc.webinterface")
+                        , settingsToggleBlock "Kodi web interface - Chorus2" "Browse and interact with your Music, Movies, TV Shows and more via a web browser. Stream music and videos on your browser. Edit and manage your Kodi media library."
+
+                        , row[ width (px 800),Background.color Colors.headerBackground, paddingXY 20 25][
+                            Input.button[Background.color Colors.cerulean, Font.color Colors.white,paddingXY 30 8, Font.size 14, Border.rounded 2]{
+                              onPress = Nothing,
+                              label = text "SAVE"
+                              }
+                            ]
+                        , el[height (px 100)](text "")
+                        ]
+            ]
         ]
     }
+
+
+settingsToggleBlock : String -> String -> Element msg
+settingsToggleBlock title description =
+  if description == "" then
+  column[ paddingEach {top = 0, bottom = 30, left = 20, right = 20}][
+    row[paddingEach {top = 0, bottom = 0, left = 0, right = 0}][
+      paragraph[width (px 275), Font.size 14, Font.color (rgb255 3 3 3), Font.medium][text title]
+    ]
+  ]
+  else
+    column[ paddingEach {top = 0, bottom = 30, left = 20, right = 20}][
+      row[paddingEach {top = 0, bottom = 20, left = 0, right = 0}][
+        paragraph[width (px 275), Font.size 14, Font.color (rgb255 3 3 3), Font.medium][text title]
+      ],
+      row[][
```

Comment:
You should be able to only add this element, according to the if statement above. That should get rid of the duplicated code above.

```
 let
        descriptionBlock =
            if description == "" then
                Element.none

            else
                row []
                    [ el [ width (px 300) ] (text "")
                    , paragraph [ width (px 400), Font.size 12, Font.color (rgb255 142 142 142) ] [ text description ]
                    ]
    in
    column [ paddingEach { top = 0, bottom = 30, left = 20, right = 20 } ]
        [ row [ paddingEach { top = 0, bottom = 20, left = 0, right = 0 } ]
            [ paragraph [ width (px 275), Font.size 14, Font.color (rgb255 3 3 3), Font.medium ] [ text title ]
            ]
        , descriptionBlock
        ]
```